### ISSUE TITLE
measureValueCompleteness thresholds changed for v5.4

### ISSUE THREAD

**Issue by don-torok** [OHDSI/DataQualityDashboard#319]
measureValueCompleteness thresholds have changed from v5.3 to 5.4.  In v5.3 the thresholds are aligned with required columns.  In v5.4 the thresholds are inconsistently applied.  They are either 100 for all rows or NULL (which I assume defaults to zero).

**Comment by katy-sadowski** [comment]
Per DQD team discussion on 12/15 - will fix this so all thresholds for this check are 100 (where required)

**Comment by katy-sadowski** [comment]
@clairblacketer this is the issue i mentioned in the meeting earlier today cc @AnthonyMolinaro

**Comment by AnthonyMolinaro** [comment]
@katy-sadowski and @clairblacketer 

The following code displays the differences between v5.3 and v5.4.  The counts are 37, 187, and 28 respectively (just remove the calls to nrow() to see the rows).  From here we can discuss how to address these.

```r
library(dplyr)
v5.3 <- read.csv("inst/csv/OMOP_CDMv5.3.1_Field_Level.csv")
v5.4 <- read.csv("inst/csv/OMOP_CDMv5.4_Field_Level.csv")

v5.3 <- v5.3[,c("cdmTableName","cdmFieldName","measureValueCompleteness","measureValueCompletenessThreshold")]
v5.4 <- v5.4[,c("cdmTableName","cdmFieldName","measureValueCompleteness","measureValueCompletenessThreshold")]

joined <- dplyr::left_join(v5.4,v5.3, by = c("cdmTableName" = "cdmTableName", "cdmFieldName" = "cdmFieldName"))
colnames(joined) <- c("table_name","column_name","is_mvc_v54","mvc_threshold_v54","is_mvc_v53","mvc_threshold_v53")

nrow(joined[which(joined$mvc_threshold_v54 != joined$mvc_threshold_v53),])
nrow(joined[which(is.na(joined$mvc_threshold_v54) & !is.na(joined$mvc_threshold_v53)),])
nrow(joined[which(!is.na(joined$mvc_threshold_v54) & is.na(joined$mvc_threshold_v53)),])
```

**Comment by clairblacketer** [comment]
Thanks @AnthonyMolinaro I think my concern is also around the columns that are used to define the not applicable checks. Do the thresholds need to be zero for that? I don't think so but I want to confirm before we make any changes

**Comment by katy-sadowski** [comment]
Thanks @AnthonyMolinaro !   There are probably 2 considerations here:

1. are we OK applying the 5.3 thresholds for the same field / same table to 5.4?
2. what should the thresholds be for new tables & fields in 5.4? (guessing maybe those also got messed up by whatever bug impacted the other thresholds)

Is there any reason you guys would be aware of that'd cause us to change the thresholds for same field/same table between versions?  And for #2, is there any convention we can follow for populating this threshold? Could it simply be 100 whenever the field is not required, and 0 when it is?

@clairblacketer for Not Applicable checks, I believe as long as the measureValueCompleteness check is run (regardless of threshold) it should be OK.  It looks like all of these checks are enabled (with `Yes` in the measureValueCompleteness column) on all threshold files.  Also if a check is missing a threshold value but is enabled, it will default to threshold 0% (see [here](https://github.com/OHDSI/DataQualityDashboard/blob/b0957c8dc3327ffe2e070d948dd12d3037d17e47/R/evaluateThresholds.R#L135)).  That said, I think it's probably a best practice to enter 0 when we want the threshold to be 0, for clarity.

**Comment by katy-sadowski** [comment]
The fix has been released in v2.4.0!
